content/hi/docs/concepts/overview/_index.md

```diff
@@ -21,75 +21,75 @@ sitemap:
+**कंटेनर डिप्लॉयमेंट दौर:** कंटेनर VMs के समान होते हैं, लेकिन उनके पास एप्लिकेशनों के बीच ऑपरेटिंग सिस्टम को साझा करने के लिए एकाकीपन की स्वतंत्रता होती है। इसलिए, कंटेनरों को हल्का माना जाता है। वीएम (VM) के समान, एक कंटेनर का अपना फाइल सिस्टम, सीपीयू का हिस्सा, मेमोरी, प्रोसेस स्पेस और बहुत कुछ होता है। क्योंकि वे अंतर्निहित अवसंरचनाओं से अलग हो गए हैं, वे क्लाउड और ऑपरेटिंग सिस्टम वितरण में पोर्टेबल हैं।
 
 कंटेनर लोकप्रिय हो गए हैं क्योंकि वे अतिरिक्त लाभ प्रदान करते हैं, जैसे:
 
-* अजाइल (Agile) एप्लिकेशन निर्माण और डिप्लॉयमेंट: VM इमेज उपयोग की तुलना में कंटेनर इमेज निर्माण की आसानी और दक्षता में वृद्धि हुई है।
-* निरंतर विकास, एकीकरण और डिप्लॉयमेंट: विश्वसनीय और लगातार कंटेनर इमेज निर्माण और त्वरित और कुशल रोलबैक के साथ तैनाती (इमेज अपरिवर्तनीयता के कारण) प्रदान करता है।
-* डेव (Dev) और ऑप्स (Ops) चिंताओं का पृथक्करण: डिप्लॉयमेंट समय के बजाय बिल्ड/रिलीज़ समय पर एप्लिकेशन कंटेनर इमेजेस (images) बनाएं, जिससे बुनियादी ढांचे से एप्लिकेशनो को अलग किया जा सके।
-* अवलोकन: न केवल ओएस-स्तर की जानकारी और मेट्रिक्स को सतह पर रखता है, बल्कि एप्लिकेशन स्वास्थ्य और अन्य संकेतों को भी लागू करता है।
-* विकास, परीक्षण और उत्पादन में पर्यावरणीय स्थिरता: लैपटॉप पर वैसे ही चलता है जैसे क्लाउड में चलता है।
-* क्लाउड और ओएस (OS) वितरण पोर्टेबिलिटी: उबंटू, RHEL, coreOS, ऑन-प्रिमाइसेस, प्रमुख सार्वजनिक क्लाउड पर और कहीं भी चलता है।
-* एप्लिकेशन-केंद्रित प्रबंधन: वर्चुअल हार्डवेयर पर OS चलाने से लेकर तार्किक संसाधनों का उपयोग करके OS पर एप्लिकेशन चलाने तक अमूर्तता के स्तर को बढ़ाता है ।
-* शिथिल युग्मित, वितरित, लोचदार, मुक्त सूक्ष्म सेवाएँ: एप्लिकेशन को छोटे, स्वतंत्र टुकड़ों में तोड़ा जाता है और उन्हें गतिशील रूप से तैनात और प्रबंधित किया जा सकता है - एक बड़ी एकल-उद्देश्य वाली मशीन पर चलने वाला एक मोनोलिथिक स्टैक नहीं।
+* Agile एप्लिकेशन निर्माण और डिप्लॉयमेंट: VM इमेज उपयोग की तुलना में कंटेनर इमेज निर्माण की आसानी और कार्यक्षमता में वृद्धि हुई है।
+* निरंतर विकास, एकीकरण और डिप्लॉयमेंट: विश्वसनीय और नियमित कंटेनर इमेज निर्माण और त्वरित और कुशल रोलबैक का समर्थन (इमेज अपरिवर्तनीयता के कारण) प्रदान करता है।
+* Dev और Ops चिंताओं का पृथकीकरण: डिप्लॉयमेंट समय के बजाय बिल्ड/रिलीज़ समय पर एप्लिकेशन कंटेनर इमेजेस (images) बनाएं, जिससे अवसंरचना से एप्लिकेशनों को अलग किया जा सके।
+* अवलोकन:यह न केवल ओएस-स्तर की जानकारी और मेट्रिक्स को सतह पर रखता है, बल्कि एप्लिकेशन के स्वास्थ्य और अन्य संकेतों को भी लागू करता है।
+* विकास, परीक्षण और उत्पादन में पर्यावरणीय संगतता: लैपटॉप पर वैसे ही चलता है जैसे क्लाउड में चलता है।
+* क्लाउड और ओएस (OS) वितरण पोर्टेबिलिटी: Ubuntu, RHEL, CoreOS, On-Premises, प्रमुख सार्वजनिक क्लाउड पर और अन्य कहीं भी चलता है।
+* एप्लिकेशन-केंद्रित प्रबंधन: वर्चुअल हार्डवेयर पर OS चलाने से लेकर तार्किक संसाधनों का उपयोग करके OS पर एप्लिकेशन चलाने तक अमूर्तता के स्तर को बढ़ाता है।
+* Loosely coupled, Distributed, Elastic-free, Microservices: एप्लिकेशन को छोटे, स्वतंत्र टुकड़ों में तोड़ा जाता है और उन्हें सक्रिय रूप से तैनात और प्रबंधित किया जा सकता है - एक बड़ी एकल-उद्देश्य वाली मशीन पर चलने वाला एक मोनोलिथिक स्टैक नहीं है।
```

Comment:
I would prefer the transliterated versions for the last line as well as the previous lines.